specification/loadtestservice/Playwright.Management/tspconfig.yaml

```diff
@@ -0,0 +1,63 @@
+emit:
```

Comment:
Are these name changes the final decision from your team to change from playwrighttesting to playwright https://github.com/Azure/azure-rest-api-specs/blob/main/specification/playwrighttesting/PlaywrightTesting.Management/tspconfig.yaml ?

Reply:
I wonder if you should follow the SDK namespace review guideline to review those new names from SDK architects https://eng.ms/docs/products/azure-developer-experience/develop/namespace-review 
@ronniegeraghty could help you proceed here.

Reply:
If the package names are changing and the new names haven't gone through the process @qiaozha linked to, then you do need to go through that process again with the new names.

Reply:
@qiaozha / @ronniegeraghty just a small heads-up to avoid any confusion, we don't want ANY change to already GA packages of loadtesting. Please help flag any errors in my PR if there're any in this respect.

Reply:
Namespace approved - https://github.com/Azure/azure-sdk/issues/8814
@ronniegeraghty @qiaozha